mlflow/utils/search_utils.py

```diff
@@ -15,9 +16,10 @@
 
 class SearchUtils(object):
     VALID_METRIC_COMPARATORS = set(['>', '>=', '!=', '=', '<', '<='])
-    VALID_PARAM_COMPARATORS = set(['!=', '='])
-    VALID_TAG_COMPARATORS = set(['!=', '='])
-    VALID_STRING_ATTRIBUTE_COMPARATORS = set(['!=', '='])
+    VALID_PARAM_COMPARATORS = set(['!=', '=', 'LIKE', 'ILIKE'])
+    VALID_TAG_COMPARATORS = set(['!=', '=', 'LIKE', 'ILIKE'])
+    VALID_STRING_ATTRIBUTE_COMPARATORS = set(['!=', '=', 'LIKE', 'ILIKE'])
+    CASE_INSENSITIVE_STRING_COMPARISON_OPERATORS = set(['LIKE', 'ILIKE'])
```

Comment:
I don't understand the meaning of `CASE_INSENSITIVE_STRING_COMPARISON_OPERATORS`. Are there non-case-insensitive operators? Could we not use the same strategy for. all `VALID_STRING_ATTRIBUTE_COMPARATORS`?